devices/Sim7080/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
```

Comment:
you can make this one a bit simpler. See for example https://github.com/nanoframework/nanoFramework.IoT.Device/blob/develop/devices/Adxl345/Properties/AssemblyInfo.cs

Reply:
Simplified